draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -178,6 +178,8 @@ Generated names reuse
 Dynamically generated names can be used to track users if used too often.
 Conversely, registering too many names will also generate useless processing.
 The proposed rule is to create and register a new generated name for a given IP address on a per execution context.
+This recommendation also mitigate a type of attacks that flood the local network
+with mDNS messages, as detailed in Section {{#security}}.
```

Comment:
Maybe we could tie it to the sentence above:
Conversely, registering too many names will also generate useless processing or flood the local network with mDNS messages, as detailed in Section {{#security}}.

Also s/execution context/browsing context/.

It is true that the proposed rule triggers this kind of attack.
It can partially be mitigated by allowing execution contexts of the same origin to share the same generated names, modulo existing browser segregation mechanisms. We could say a word about this in the security section.
In any case, data URL frames can be created, so we will probably need a registration throttling mechanism.

Reply:
I have added the attack by adding remote candidates and flooding via queries. I feel we probably would not need the throttling on adding remote candidates, and instead we could rely on the throttling/rate control of the mDNS messages, which mitigates several DoS attacks via the same mechanism.